### ISSUE TITLE
Remove "a11y" from "Accessibility on VA.gov" page

### ISSUE THREAD

**Issue by coforma-jamie** [department-of-veterans-affairs/va.gov-team#93877]
### Description
Hi Platform Content team! We're working to remove mentions of the word "A11y" from our accessibility documentation (because the word itself, as a somewhat obtuse acronym, isn't accessible in itself 😛)

Can you change the following:

Under "Improve your skills"

> A11y Champs
> Accessibility Champtions ("a11y champs") is a grassroots, volunteer-led educational program. Anyone working at the VA, with access to the DSVA slack channel, can become an accessibility champion through this curriculum!
> [Start your A11y Champs journey.](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/teams/vsa/accessibility/a11y-champs)

this should be:

> Accessibility Champs
> Accessibility Champions is a grassroots, volunteer-led educational program. Anyone working at the VA, with access to the DSVA slack channel, can become an accessibility champion through this curriculum!
> 
> [Start your Accessibility Champs journey.](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/teams/vsa/accessibility/a11y-champs)

Under "About the Audit"

> - This page was created as a result of a content audit of a11y documentation found across multiple environments.
> - It was then analyzed and aggregated to reflect current a11y practices.

this should be:

> - This page was created as a result of a content audit of accessibility documentation found across multiple environments.
> - It was then analyzed and aggregated to reflect current accessibility practices.

### Relevant URLs
https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-on-va-gov

### Which type of team are you on? (Platform team, VFS team, or Leadership)
CAIA

**Comment by jknipes** [comment]
Hi @coforma-jamie , thanks for your content suggestions. The Platform Content team will review your request and put it in our backlog to refine for next steps.

**Comment by jknipes** [comment]
work is completed, we are closing this ticket
